### PR TITLE
Xenobio fix/nerf

### PR DESCRIPTION
**Pull request by RadiantFlash7**
## About The Pull Request
Comments out bluespace and red slime in cross breeding.

## Why It's Good For The Game
It was overlooked and should have been already commented out.

## Pre-Merge Checklist

- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.

## Changelog

:cl:
tweak: commented out xenobioslime cross breeding for bluespace and red slimes.
/:cl:

### CODE REVIEW

File: code/modules/research/xenobiology/crossbreeding/stabilized.dm [hunk 1]
```diff
@@ -84,10 +84,10 @@ Stabilized extracts:
 
 /obj/item/slimecross/stabilized/pyrite
 	colour = "pyrite"
-
+/*
```
Comment (ProbablyCarl) [inline code/modules/research/xenobiology/crossbreeding/stabilized.dm:87]: Not needed.

Comment (RadiantFlash7) [reply code/modules/research/xenobiology/crossbreeding/stabilized.dm:87]: I was told having it in your back removed your slowdown. Which is the reason we were yeeting the red slime extra across the board?

Comment (ProbablyCarl) [reply code/modules/research/xenobiology/crossbreeding/stabilized.dm:87]: Yet you left in the bluespace extract?

Comment (RadiantFlash7) [reply code/modules/research/xenobiology/crossbreeding/stabilized.dm:87]: The bluespace extract when in a backpack, teleports someone randomly within sight from my understanding.
It isn't an actively usable item like the regenerative bluespace extract where you can use it on someone to teleport them.
If you would like, I can comment that out as well. But It wasn't contributing to the same problem that the other one was being commented out for, like the stabilized red extract is.

**Pull request merged**